src/events/message.js

```diff
@@ -1,3 +1,9 @@
+const { emojis } = require('../constants');
+
 exports.run = (bot, message) => {
     bot.commands.handleCommand(message);
+
+    if (message.channel.id === bot.config.introductionsChannel) {
+        message.react(emojis.ct_semicolon);
+    }
```

Comment:
I'd rather have this outsourced in the commands manager, make a separate class method e.g. `handleMessage` or `handleReactions` and call that from here, and before the Commands#handleCommand call.